CONTRIBUTING.md

```diff
@@ -40,12 +40,60 @@ You'll need manual configuration to make the application running:
 
 - Add `postcss.config.js` and `tailwind.config.js` at the root of your project if you want to use TailwindCSS.
 
-- If you want to an **application configuration**, follow the following steps:
+- Disable the properties `noPropertyAccessFromIndexSignature` / `noImplicitOverride` in `tsconfig.json`:
 
-  1. Add the config file as asset for the application in `angular.json`:
+  ```json
+  "compilerOptions": {
+    ...
+    "noImplicitOverride": false,
```

Comment:
Could you explain why you had to change those values ?
Are there some errors in your app ? Or do you think these options should be a standard ?

Note that all app/lib tsconfig files extend the root one, so it may be appropriated to add this rules in the root one instead (if opportune).

Reply:
For me the `tsconfig.json` contained these 2 values:

```
    "noImplicitOverride": true,
    "noPropertyAccessFromIndexSignature": true,
```

With `"noImplicitOverride": true`, caused these type of errors in the build:

```
Error: libs/feature/search/src/lib/utils/service/fields.ts:96:3 - error TS4114: This member must have an 'override' modifier because it overrides a member in the base class 'SimpleSearchField'.

96   getAvailableValues(): Observable<FieldAvailableValue[]> {
     ~~~~~~~~~~~~~~~~~~
```

With `"noPropertyAccessFromIndexSignature": true`, caused these type of errors in the build:

```
Error: libs/feature/catalog/src/lib/organisations/organisations.service.ts:132:25 - error TS4111: Property 'eng' comes from an index signature, so it must be accessed with ['eng'].

132             group.label.eng ? group.label.eng : group.name,

```
Maybe previous versions didn't add that values to `tsconfig.json`. But at least with my setup were causing issues. I had to set to `false` to work. I see also removing them works, so probably something has change and these properties were not added previously.

Reply:
Thanks @josegar74 for explanations.
Just tested, it adds the following
```json
"compilerOptions": {
  "useDefineForClassFields": false,
  "noImplicitOverride": true,
  "noPropertyAccessFromIndexSignature": true,
},
"angularCompilerOptions": {
  "enableI18nLegacyMessageIdFormat": false,
}
```
You should just remove them instead of setting them to `false`, we should avoid to have useless properties which overrides defaults, for maintenance sake.

Actually, nx generator changes over time, tsconfig, jest config etc... so it's not a big deal. Our work it to try to keep consistency and to have good options for all these configs.
That said, I think that `noImplicitOverride` could be a correct option to add, we can see that later.

Can you update your PR with telling in the doc to remove the options instead of putting them to `false`, thanks !